Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -159,6 +164,43 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._repeatCountElement.textContent = Number.abbreviate(count);
     }
 
+    set timestamp(timestamp)
+    {
+        this._message._timestamp = timestamp;
+        if (this._element)
+            this._renderTimestamp(true);
+    }
+
+    _renderTimestamp(renderlatestTimestamp = false)
```

Comment:
Style: names should be camel case, e.g. `renderLatestesTimestamp`. Also a falsey value is the default if the parameter is not provided, so no need for `= false`.